src/Microsoft.DotNet.Cli.Utils/CommandResolution/PackagedCommandSpecFactoryWithCliRuntime.cs

```diff
@@ -21,9 +21,41 @@ private static void AddAditionalParameters(string commandPath, IList<string> arg
         {
             if(PrefersCliRuntime(commandPath))
             {
-                arguments.Add("--fx-version");
-                arguments.Add(new Muxer().SharedFxVersion);
+                var runtimeConfigFile = Path.ChangeExtension(commandPath, FileNameSuffixes.RuntimeConfigJson);
```

Comment:
Yeah, I chatted about this with @eerhardt and it turns out that NuGet pack includes the runtimeconfig file by default.